README.md

```diff
@@ -51,6 +51,11 @@ func main() {
     myLogger.Log("CustomLevelName", "EventName", "message...", map[string]string {
       "key": "value"
     })
+
+    // Level filtering (define per transport!)
+    filteredTransport := logger.NewTransport( ... )
+    filteredTransport.filter = logger.FilterByMinimumLevel(logger.NewLevelFilter("Warning"))
```

Comment:
is it also possible to do filtering by Event?

If I am using the filter, I may want to check against the `Entry` object, and then the filter function can do checks like `entry.Level == "Warning"`?